### DISCUSSION TITLE
Prisma has a very slow startup time when my database has 100+ tables 😥

### DISCUSSION THREAD

**Discussion by meotimdihia** [prisma/prisma#10232] [Q&A] [answered]
I am using `ts-node-dev` to run Fastify in the development environment.
when I modified my application code => my application restarted, and it took 10s to complete because of  Prisma.

This problem is annoying, it is affecting development speed. 😥
Do we have a way to solve this problem - something like separate Prisma with the application...

https://user-images.githubusercontent.com/300961/141484875-8f885337-583f-4a4c-98cc-7814d0aabdbb.mp4

**Comment by austincrim** [comment, marked as answer]
Hi @meotimdihia! 

If you are certain this is due to Prisma, please open a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with as much context about your environment as possible; our engineering teams will pick it up from there. 

This definitely looks frustrating so we would love to help get it figured out!

**Reply by meotimdihia** [reply]
> Hi @meotimdihia!
> 
> If you are certain this is due to Prisma, please open a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with as much context about your environment as possible; our engineering teams will pick it up from there.
> 
> This definitely looks frustrating so we would love to help get it figured out!

I am 99% sure because of reasons: 
- Prisma is slow with Next.js. I have to remove Prisma from next.js Prisma and separate it from the API server.
- I have tested by removing all Prisma related in my API code and my application became fast again.

I think it is my PC is not very high performance but even with high spec PC, I doubt it still takes 3-5s to start.

I will create an issue about this problem.

**Reply by meotimdihia** [reply]
Sorry, I have found out the problem by enabling debug log. This happened because Sequelize (that I used in past time) when I included a Prisma lib.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
